schemas/environment.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from 'zod';
+import { zfd } from 'zod-form-data';
+
+// Custom parser for UPLOADTHING_TOKEN to remove token name and quotes
+const parseUploadThingToken = (token: string) => {
+  return token.replace(/^(UPLOADTHING_TOKEN=)?['"]?|['"]$/g, '').trim();
+};
+
+export const createUploadThingTokenForm = z.object({
```

Comment:
Although this _is_ used in a form context, it doesn't use formData (which the schema below does) so the naming is confusing.

This should be `createUploadThingTokenSchema`.